### PR TITLE
Millestone #1: WebPack and first layout

### PR DESCRIPTION
**Pull request by GuilhermeGiachellin**
## What was done in this Milestone:

- Add WebPack to the repo.
- Created a simple layout.
- Array of objects that contains the information of the list.
- All the list is generated via JS.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -5,8 +5,9 @@ module.exports = {
   entry: './src/index.js',
   mode: 'development',
   output: {
-    filename: 'bundle.js',
+    filename: 'index.bundle.js',
     path: path.resolve(__dirname, 'dist'),
+    clean: true,
   },
   plugins: [
     new HtmlWebpackPlugin({
```
Comment (teekaytech) [inline webpack.config.js]: - [x] You need to configure the basic commands for building Webpack in development and production mode using the `package.json` file. I believe you've done something similar in your Webpack exercise. We should be able to set up and run Webpack devserver with no errors, using the configuration. At the moment, package.json is completely missing in your project. Kindly fix this.

File: src/index.js [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
-    description: 'clean the floor',
-    completed: false, 
-}
+  description: 'clean the floor',
+  completed: false,
+  index: 3,
+},
 ];
-
+console.log(list.length);
```
Comment (teekaytech) [inline src/index.js]: - [x] Kindly remove this console statement: they make your code look messy. Always double-check to ensure that you remove all console log statements in the future before you commit your changes.

File: src/style.css [hunk 3]
```diff
@@ -1,4 +1,59 @@
...
+  height: 50px;
+  border: 0.5px solid #d4d4d4;
+}
+
+::placeholder {
+  font-style: italic;
+  font-size: 1em;
+  color: #d4d4d4;
+  padding-left: 1vw;
+}
```
Comment (teekaytech) [inline src/style.css]: - [x] Also, you should ensure that the CSS created is used to style the list using the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). I can see that you've done this to a reasonable extent. However, you have to align your booklist to the center of the page, as we have it in the video.

File: README.md [hunk 4]
```diff
@@ -5,10 +5,10 @@
...
 ## Image Preview
+![image](https://user-images.githubusercontent.com/81584449/125695914-8999bca3-2cc6-49bc-97d3-04b5d95bb285.png)
 
-
-##  Live Demo
-
+<!-- //##  Live Demo
+[Demo](https://guilhermegiachellin.github.io/To-Do-List/)-->
 
 ## What was built in this project:
```
Comment (teekaytech) [inline README.md:13]: **OPTIONAL**

- [x] Kindly update your Readme file
- I strongly suggest you add the `Getting Started` section, where you state the instructions and code samples that will be needed to clone your project, install its dependencies, and starting the project on a dev environment.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @GuilhermeGiachellin :wave:

_Tis milestone is almost complete as most of the tasks have been implemented :handshake:_

- Your PR title, summary and commit messages looks okay :ok_hand: 
- All linter checks are running with no errors :100: 

__However, you need to make some required changes before moving on to the next assignment.__
> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes** :recycle: 
Check the comments under the review.

### **Optional Changes** :zap: 
_Every comment with the **OPTIONAL** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @GuilhermeGiachellin🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**